web_src/js/components/PullRequestMergeForm.vue

```diff
@@ -1,9 +1,17 @@
 <template>
+  <!--
+  if this component is shown, either the user is admin (can do merge without checks), or they is a writer who has the permission to do merge
+  if the user is a writer and can't do merge now (canMergeNow==false), then only show the Auto Merge for them
+  -->
   <div>
+    <!-- eslint-disable -->
+    <div v-if="mergeForm.hasPendingPullRequestMerge" v-html="mergeForm.hasPendingPullRequestMergeTip" class="ui info message" style="margin-top: 10px;"></div>
```

Comment:
Answer the `eslint-disable` ahead: at the moment `hasPendingPullRequestMergeTip` is generated as HTML content by template, so the `v-html` must be used here.